### PR TITLE
Add a preloader to team page

### LINKED ISSUE #1053: Add a preloader to the Team page
**Is your feature request related to a problem? Please describe.**
The team page is taking some time to load the data to the page. It shows as in below for few seconds

Screenshot from 2021-07-27 12-09-23
**Describe the solution you'd like**
Add a preloader to the team page to avoid that

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose

The purpose of this PR is to fix #1053

## Goals
To add skeleton loading profiles until the profiles are loaded

## Approach
Used skeleton loading profiles in the relevant sections 

### Screenshots
Screenshot from 2021-08-09 09-01-16

  
### Preview Link

https://pr-1077-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by janethavi — DISMISSED** [review]
Superb @gajindubandara! You have done a very good job 💯

File: team.html [hunk 1]
```diff
@@ -163,6 +413,35 @@ <h5 class="title">
...
+            document.getElementById("skeletonDevTeam").style.display = "none";
+            document.getElementById("skeletonAlumni").style.display = "none";
+            document.getElementById("advisoryPanel").style.display = "block";
+            document.getElementById("externalCollaborators").style.display = "block";
+            document.getElementById("executiveCommitee").style.display = "block";
+            document.getElementById("techTeam").style.display = "block";
+            document.getElementById("alumni").style.display = "block";
+        }
+    }
+
```
Comment (jayasanka-sack) [inline team.html]: I don't think that we need this mechanism. Just include them inside the desired div itself. Moustache will replace the content.

ex: 
```
<div id="alumni">
   // skeleton goes here
</div>
```

Any thoughts @Gravewalker666?

---------------
btw,  
`document.getElementById("alumni").style.display = "none";` is equal to `$('#alumni').hide();`

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: team.html (hunk 1 above)
Comment (Gravewalker666) [reply team.html]: Yup, I agree

Comment (Gravewalker666) [reply team.html]: @gajindubandara take a look at this 
https://github.com/Gravewalker666/grave-web/blob/22c44a254d6f47f4cdec03af96d82466a6cb78d1/src/index.js#L67-L86

Comment (Gravewalker666) [reply team.html]: Corresponding html, 
https://github.com/Gravewalker666/grave-web/blob/22c44a254d6f47f4cdec03af96d82466a6cb78d1/src/index.html#L237-L251

File: assets/css/global-custom.css [hunk 2]
```diff
@@ -110,6 +110,70 @@ body {
...
+    justify-content: center;
+    margin-bottom: 48px;
+}
+
+.skeleton-row {
+
+    justify-content: center;
+    max-width: 150px;margin-top: 58px;
+
+}
```
Comment (jayasanka-sack) [inline assets/css/global-custom.css]: fix formatting

File: assets/css/global-custom.css [hunk 3]
```diff
@@ -110,6 +110,70 @@ body {
...
+    margin-bottom: 10px;
+    animation: pulse 2s infinite ease-in-out;
+}
+
+.skeleton-lineTwo{
+    width: 150px;
+    height: 15px;
+    margin-bottom: 10px;
+    animation: pulse 2s infinite ease-in-out;
+}
```
Comment (jayasanka-sack) [inline assets/css/global-custom.css]: only one property changes here. use a common class

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
https://code.tutsplus.com/tutorials/30-css-best-practices-for-beginners--net-6741

File: assets/css/global-custom.css [hunk 4]
```diff
@@ -110,6 +110,66 @@ body {
...
+
+.skeleton .line {
+    width: 120px;
+    height: 15px;
+    margin-bottom: 10px;
+    animation: pulse 2s infinite ease-in-out;
+}
+
+.skeleton .line-two {
```
Comment (jayasanka-sack) [inline assets/css/global-custom.css]:
```suggestion
.skeleton .line-secondary {
```

File: assets/css/global-custom.css [hunk 5]
```diff
@@ -110,6 +110,66 @@ body {
...
 
 /*end of upspeak testimonial section*/
+/*Skeleton preloader section*/
+.skeleton {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 48px;
+}
+
+.skeleton .text {
```
Comment (jayasanka-sack) [inline assets/css/global-custom.css]: are there texts in skeleton?

File: assets/css/global-custom.css [hunk 6]
```diff
@@ -110,6 +110,66 @@ body {
...
+    height: 15px;
+    margin-bottom: 10px;
+    animation: pulse 2s infinite ease-in-out;
+}
+
+.skeleton .line-two {
+    width: 150px;
+}
+
+.skeleton .img {
```
Comment (jayasanka-sack) [inline assets/css/global-custom.css]:
```suggestion
.skeleton .avatar {
```

File: assets/css/global-custom.css (hunk 5 above)
Comment (gajindubandara) [reply assets/css/global-custom.css]: yes!

File: team.html [hunk 7]
```diff
@@ -52,10 +101,59 @@ <h1 class="display-3 text-center mb-5" >Advisory Panel</h1>
...
-
+        <h1 class="display-3 text-center mb-5">External Collaborators</h1>
         <!--Profile cards goes here-->
-        <div id="externalCollaborators"></div>
+        <div id="externalCollaborators">
+            <!-- Skeleton Profiles goes here -->
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-3 skeleton">
+                        <div>
```
Comment (jayasanka-sack) [inline team.html]:
```suggestion
                    <div class="col-md-3">
                        <div class="skeleton">
```

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @gajindubandara!

**Pull request merged**